source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -421,6 +371,43 @@ This example specifies the following driver behavior in a pool of
    :emphasize-lines: 3-5
    :dedent:
 
+.. _mcs-logger-settings:
+
+Logger Settings
+---------------
+
+Chain the `applyToLoggerSettings() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.Builder.html#applyToLoggerSettings(com.mongodb.Block)>`__
+method to modify the logging behavior of the driver.
+
+The following table describes all the methods you can chain to your
+settings to modify the driver's logger behavior:
+
+.. list-table::
+   :widths: 40 60
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``maxDocumentLength()``
+     - Sets the maximum document length, in characters, of a single log
+       message. The default value is 1000 characters.
```

Comment:
Comment: This is the only setting on this page that provides the default setting. Do you know if the other settings have default values? If so, you should make a call on how to standardize the setting descriptions throughout this page to include/not include the default